packages/angular/projects/ui-angular/src/lib/primitives-poc/button/button-component/button.component.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+  OnInit,
+  ViewEncapsulation,
+} from '@angular/core';
+
+@Component({
+  selector: 'button[amplify-button-poc]',
+  templateUrl: './button.component.html',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AmplifyButtonComponent implements OnInit {
+  @Input() type: 'submit' | 'button' = 'button';
+  @Input() fullWidth: boolean | string = false;
+  @Input() isDisabled: boolean | string = false;
```

Comment:
Let's have the type scoped to just `boolean` instead of `boolean | string`.

Reply:
changed.